util/signal.cc

```diff
@@ -206,7 +206,12 @@ SignalThread::SignalThread(bool exit_on_signal_arg) :
   sigaddset(&set, SIGUSR2);
 
   strcpy(lock_name, "/XXXXXXXXX");
-  mktemp(lock_name);
+  // random lock_name required
+  if (mkstemp(lock_name) != -1)
+   {
```

Comment:
Braces doesn't appear to line up properly here. The open brace should be under the "i" in "if". Double check indentation on the next several lines as well.

Reply:
`mkstemp` based approach is removed. The braces also. 
Unfortunately Travis CI tests failing with new random string based approach. They are running fine locally.